src/components/shared/DatePicker.tsx

```diff
@@ -29,14 +29,12 @@ function DatePickerCustomInput({ onClick, onChange, value }: DatePickerCustomInp
 
 function DatePick({ currentDate, onChange }: DatePickType) {
   return (
-    <div className="w-72 h-11 m-20">
```

Comment:
Thank you to removed this mistaken css style :D

Reply:
no problem, you are welcome